js/views/transactionModalVw.js

```diff
@@ -43,6 +43,9 @@ module.exports = baseVw.extend({
     'click .js-transactionShowContract': 'showContract',
     'click .js-transactionHideContract': 'hideContract',
     'click .js-acceptResolution': 'acceptResolution',
+    //'click .js-refundTransaction': 'showRefundOrder',
```

Comment:
do you mean for these commented lines to remain in? there's also one in refundOrder().